database/schema.sql

```diff
@@ -9,10 +9,19 @@ DROP TABLE tag CASCADE;
 DROP TABLE game CASCADE;
 DROP TABLE platform_assignment CASCADE;
 DROP TABLE game_tag_matching CASCADE;
+DROP TABLE subscriber
```

Comment:
Is this meant to be in this pull request?

Reply:
No it's not, good spot I will try to fix this!

Reply:
Looks good to me